soccer/radio/USBRadio.cpp

```diff
@@ -237,11 +237,11 @@ void USBRadio::send(Packet::RadioTx& packet) {
 
         forward_packet[offset++] = (dribbler & 0xf0) | (robot_id & 0x0f);
         forward_packet[offset++] = kick;
-        forward_packet[offset++] = robot.use_chipper() |
-                                   (robot.kick_immediate() << 1) |
-                                   (robot.sing() << 2) | (robot.anthem() << 3);
-        forward_packet[offset++] = robot.accel();
-        forward_packet[offset++] = robot.decel();
+        forward_packet[offset++] = (robot.shootmode() == Packet::Control::CHIP) |
+                                   ((robot.triggermode() == Packet::Control::IMMEDIATE) << 1) |
+                                   (robot.song() << 2) | (0 /*robot.anthem()*/ << 3);
+        forward_packet[offset++] = 10; //robot.accel();
+        forward_packet[offset++] = 10; //robot.decel();
     }
```

Comment:
It would be nice if we could find a way to use the same serialization/deserialization code in firmware and soccer.  It may be hard to do though.